<?xml version="1.0"?>
<launch>
  <arg name="reserve_area" default="0.03" />
  <arg name="ar_height" default="0.1" />
  <arg name="source_frame" default="/photoneo_center_optical_frame" />
  <arg name="target_frame" default="/base_link" />

    <node pkg="oneshot_calibration_system" type="workspace_pointcloud_extractor" name="workspace_pointcloud_extractor" output="screen">
      <param name="ar_height"    value="$(arg ar_height)" />
      <param name="reserve_area" value="$(arg reserve_area)" />
      <param name="target_frame" value="$(arg target_frame)" />
      <param name="source_frame" value="$(arg source_frame)" />

      <rosparam file="$(find oneshot_calibration_system)/config/oneshot_calibration_config.yaml" command="load" />

      <remap from="input_pointcloud" to="/photoneo_center/pointcloud" />
      <remap from="output_pointcloud" to="/workspace_pointcloud" />

  </node>
</launch>
